script/build

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+'use strict';
+
+let rollup = require('rollup').rollup;
+let babel = require('rollup-plugin-babel');
+let babelrc = require('babelrc-rollup').default;
+let filesize = require('file-size');
+let gzipSize = require('gzip-size').sync;
+let stat = require('fs').statSync;
+let minify = require('uglify-js').minify;
+let pkg = require('../package.json');
+
+rollup({
```

Comment:
> The duplication of work point is fair, though I wonder if we should just fix rollup/rollup#655 instead?

I commented there how I think we should do it.

> I do like the logging at the end of the build script, perhaps we should add that to the CLI

Yeah, though I'm not sure what to do about the min+gzip stuff. The minifier I used doesn't work with ES6 (uglify-js).